aio_pika/channel.py

```diff
@@ -50,17 +50,17 @@ def __init__(self, connection, channel_number: int = None,
         self.loop = connection.loop
 
         self._connection = connection
-        self._done_callbacks = CallbackCollection()
-        self._return_callbacks = CallbackCollection()
-        self._channel = None  # type: aiormq.Channel
+        self._done_callbacks = CallbackCollection(self)
+        self._return_callbacks = CallbackCollection(self)
+        self._channel: Optional[aiormq.Channel] = None
```

Comment:
This syntax not supported for Python 3.5:

```
$ python3.5
Python 3.5.4 (v3.5.4:3f56838976, Aug  7 2017, 12:56:33)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> f: bool = True
  File "<stdin>", line 1
    f: bool = True
     ^
SyntaxError: invalid syntax
>>>
```

Reply:
If revert then pycharm highlight that the wrong type. If write `type: Optional[aiormq.Channel]` then DeepSource [will swear](https://deepsource.io/gh/mosquito/aio-pika/run/42d515a2-ebae-43eb-8f28-6cc8c6973a25/python/PYL-C0412). What to do?